processor/metricstransformprocessor/datapoint_aggregation.go

```diff
@@ -49,6 +49,15 @@ func (mtp *metricsTransformProcessor) mergeTimeseries(groupedTimeseries map[stri
 	return newTimeSeriesList
 }
 
+// sortTimeseries performs an in place sort of a list of timeseries by start timestamp
+// Returns the sorted timeseries
+func (mtp *metricsTransformProcessor) sortTimeseries(timeseries []*metricspb.TimeSeries) []*metricspb.TimeSeries {
+	sort.Slice(timeseries, func(i, j int) bool {
+		return mtp.compareTimestamps(timeseries[i].StartTimestamp, timeseries[j].StartTimestamp)
+	})
+	return timeseries
```

Comment:
Is that actually useful to return a slice in this function, given that sorting actually happens in place (i.e. input is modified)? It might be confusing for a function to return the mutated input.

Reply:
It's useful but yes I agree, probably misleading. Updated to not return anything.